src/app.ts

```diff
@@ -2,14 +2,16 @@
 
 import Bugsnag from '@bugsnag/react-native'
 import { asObject, asString } from 'cleaners'
-import { Text, TextInput } from 'react-native'
+import { LogBox, Text, TextInput } from 'react-native'
 import RNFS from 'react-native-fs'
 
 import { changeTheme, getTheme } from './components/services/ThemeContext'
 import { ENV } from './env'
 import { NumberMap } from './types/types'
 import { log, logToServer } from './util/logger'
 
+LogBox.ignoreAllLogs()
```

Comment:
Nope. We should show annoying log boxes. I think this was a mistake?

Reply:
Oops. that snuck in there.